thunder/torch/__init__.py

```diff
@@ -1208,6 +1208,12 @@ def relu6(a: TensorProxy, /, inplace: bool = False) -> TensorLike:
     return clamp(a, 0, 6)
 
 
+@torchsymbol(torch.nn.functional.hardswish, id="torch.hardswish", is_method=False)
+def hardswish(a: TensorProxy, /, inplace: bool = False) -> TensorLike:
```

Comment:
This looks pretty good; following the above comment, please also add a check that `a` has a floating point dtype